meeting-minutes/2025/2025-04-16.md

```diff
@@ -0,0 +1,197 @@
+    - With majory vote, the paper will proceed to be published.
+    - Omar will take the action item to convert current state of whitepaper to the OASIS template.
+
+## 3.2 Issues / Pull Requests
+
+- PR [#73](https://github.com/oasis-tcs/openeox/pull/73)
+    - Omar and Langley discuss the use of "EoSecS".
+    - Langley suggests not using acronyms at all to avoid conflict.
+    - Thomas Schmidt and Rogue disagree and emphasize the need for acronyms as long as they are clearly definined in the standard.
+        - Acronym duplication and conflict is not a new thing. 
+    - Thomas Schmidt suggests voting between the three most popular options:
+      - EOSS
+      - EoSecS
+      - EOSSec
+    - TC majority vote is for EoSSec.
+    - Thomas Schmidt set motion to accept the End of Security Support Acronym as EoSSec.
+    - Omar and Rogue seconded. No discussion. No objections. Motion passed.
+    - Thomas Schmidt additionally set motion to accept the End of Security Support Acronym definition as stated in PR [#73](https://github.com/oasis-tcs/openeox/pull/73). T
+    - homas Schaffer seconded. No discussion. No objections. Motion passed.
+- PR [#75](https://github.com/oasis-tcs/openeox/pull/75)
+    -  Thomas Schmidt presents the proposed definition for General Availability (GA).
+    -  No comments or discussion.
+    -  Thomas Schmidt sets the motion to accept the General Availability (GA) definition in PR [#75](https://github.com/oasis-tcs/openeox/pull/75)
+    -  Thomas Shaffer seconded. No discussion. No objections. Motion passed.
+- PR [#78](https://github.com/oasis-tcs/openeox/pull/78)
+    - Thomas Schmidt presents the proposed definition for "product".
+    - Thomas Schaffer asks whether the term "services" should be changed to singular "service".
+    - Thomas Schmidt sets the motion to accept the defintion of "product" with the amendment of "services" to "service" as stated in PR[#78](https://github.com/oasis-tcs/openeox/pull/78).
+    - Omar and Rogue seconded. No discussion. No objections. Motion passed.
+- PR [#80](https://github.com/oasis-tcs/openeox/pull/80)
+    -  Thomas Schmidt presents the modified "End of Sales" definition with the amendment of striking the mention of version and license.
+    -  Thomas Schmidt sets the motion to accept the updated "End of Sales" definition as stated in PR [#80](https://github.com/oasis-tcs/openeox/pull/80).
+    -  Thomas Schaffer seconded. No discussion. No objections. Motion passed.
+- PR [#81](https://github.com/oasis-tcs/openeox/pull/81)
+    - Thomas Schmidt presents the updated definition for "product lifecycle".
+    - Thomas Schmidt sets the motion to accept the updated "product lifecycle" definition as stated in PR [#80](https://github.com/oasis-tcs/openeox/pull/81).
+    - Rogue seconded. No discussion. No objections. Motion passed.
+- PR [#10](https://github.com/oasis-tcs/openeox/pull/10)
+    - Thomas Schmidt presents the Core Schema draft PR and discusses .
```

Comment:
```suggestion
    - Thomas Schmidt presents the Core Schema draft PR and discusses it.

Reply:
I would rather drop the "and discusses " part than have Thoms publicly discuss his own PR with himself 🙈

Reply:
Also fine with me.

```suggestion
    - Thomas Schmidt presents the Core Schema draft PR.